### ISSUE TITLE
Archive - No results, Nothing Found Message not appearing

### ISSUE THREAD

**Issue by nmadams3** [dudaster/ele-custom-skin#14]
After creating a loop template and applying the custom skin/template to the archives, everything works as it should when searching _except_ when there are no results.  No message appears despite Elementor's Advanced > No Message Found area having content.  If I change the skin to cards or classic, the message will appear.

**Comment by Artneo7** [comment]
I'm looking for this feature too. Any plan to implement it? Thanks!

**Comment by bmarsilio** [comment]
I don't know if it is right, but works :)

https://wordpress.org/support/topic/dont-show-nothing-found-message/

**Comment by formulaclick** [comment]
Same problem here. If I use Classic Skin, then "Nothing found" message is shown, but if I use a custom skin, then nothing is rendered when there are no posts. This a bug and should be fixed! Thanks @dudaster

**Comment by formulaclick** [comment]
@dudaster Any news on this? As Pro user this should be fixed. I think it is not complicated to solve! Thanks

**Comment by dudaster** [comment]
Next week I’ll release an update.

**Comment by acarravedo** [comment]
@dudaster Hi, excuse me, how is this update?

**Comment by dudaster** [comment]
It should have the stuff. Have you tried it?

**Comment by acarravedo** [comment]
@dudaster 
Yes, sorry, we had not seen the last update of the plugin, we have them disabled ... all OK.
Thank you very much.

**Comment by graphicfixations** [comment]
Hi, I updated but still experiencing this same issue. Nothing shows up at all if there are no matches if I am using a custom skin. Please help!

**Comment by dudaster** [comment]
Does it show with classic skin?

**Comment by capstonedesign** [comment]
Hello. Was anything done to fix this since then? I am not seeing any message when there are no posts to show from the query.

**Comment by MayurAtThem** [comment]
**SOLUTION:** Add the "post-archive-no-records-msg" class to the post archive widget and below CSS should add the no records found message.

`
.post-archive-no-records-msg .elementor-posts-nothing-found:before {
    content: "No records found";
    color: #474B55;
    font-size: 14px;
}
`
